dev-db/sqliteman/sqliteman-1.2.2-r3.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```

Comment:
Bumped, requested changes done.
No change in installed files, repoman approves.